Experiment

altale opened the pull request:
The experiment results based on a smaller dataset.

altale committed:
Experiment

hzhwcmhf commented on Readme.md:
```diff
@@ -144,6 +144,36 @@ gen:    i don' t know .
+|   175   |   175   |     √     |    0.0001     |   0.2    |   samplek   |     97.704     |     102.851     |
+|   175   |   175   |     ×     |    0.0001     |   0.1    |   samplek   |     90.149     |     95.310      |
+|   175   |   175   |     ×     |    0.0001     |   0.3    |   samplek   |     89.750     |     95.042      |
+|   175   |   175   |     ×     |    0.0005     |   0.2    |   samplek   |     88.688     |     93.421      |
```
Reorganize the table, with the best on the top.   Use **bold** to mark the difference between the best and this line

hzhwcmhf replied:
Give your command to reproduce the best experiment

hzhwcmhf commented on Readme.md:
```diff
@@ -144,6 +144,36 @@ gen:    i don' t know .
+
+We can consider the parameters used in the 6th experiment are relatively best.  So we did another five experiments based on the best parameters.
+
+|        seed        | dev perplexity | test perplexity |
```
add bleu

altale committed:
Add more information to README

1. Reorganize the table, with the best on the top. Use bold to mark the difference between the best and this line.
2. Give my command to reproduce the best experiment.
3. Add bleu

altale committed:
Fix bold bug

On the github page, we cannot be able to add bold to some signals such as √ and ×. Use yes and no instead.

altale committed:
Add learning rate to arguments

hzhwcmhf commented on Readme.md:
```diff
@@ -144,6 +144,50 @@ gen:    i don' t know .
+
+We can consider the parameters used in the first experiment are relatively best. 
+
+To reproduce the best experiment, you should use `cotk` to load the dataset `OpenSubtitles_small` first. After that, change the learning rate in `run.py` to `1e-4` and then run the following command to train the model:
```
Change this

altale committed:
Add experiment result of universal dataset
